monitoring/alerts/scheduledQueries/dev/Burst-Header-Failures.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "Burst-Header-Failures",
+    "location": "${Ngsa_Alert_Location}",
+    "properties": {
+        "displayName": "Burst-Header-Failures",
+        "description": "Burst Header not returned on > 20 requests to NGSA Cosmos",
+        "severity": 3,
+        "enabled": true,
+        "evaluationFrequency": "PT10M",
+        "windowSize": "PT15M",
+        "autoMitigate": false,
+        "scopes": [
+            "/subscriptions/${Ngsa_Subscription_Guid}/resourceGroups/${Ngsa_Common_Services_RG}/providers/Microsoft.OperationalInsights/workspaces/${Ngsa_Log_Analytics_Name}"
+        ],
+        "criteria": {
+            "allOf": [
+                {
+                    "query": "ingress_CL | where k_app_s == 'ngsa-cosmos' and x_load_feedback_s == '' and user_agent_s startswith 'l8r' and Zone_s == 'westus2' | summarize FailedRequests=count() by bin(TimeGenerated, 5m), Zone_s | project-rename AggregatedValue=FailedRequests",
```

Comment:
Wondering if `westus2` should be used as a constant string in query.

Reply:
Never mind, just remembered that burst metrics is only deployed in WestUS2 cluster.
Instead of using a constant string we might put westus2 under values on Line 25.
But the PR LGTM.